pkg/iac/scanners/terraform/scanner_test.go

```diff
@@ -1119,3 +1119,99 @@ resource "aws_s3_bucket" "test" {
 		})
 	}
 }
+
+func Test_RenderedCauseShouldNotPanicOnUnknownValues(t *testing.T) {
```

Comment:
Perhaps we can merge this test into:
https://github.com/aquasecurity/trivy/blob/0e7ff449c3797d4a78dcbbf654f6aa27d569c441/pkg/iac/scanners/terraform/scanner_test.go#L998

Reply:
Yes good call - updated: https://github.com/aquasecurity/trivy/pull/8604/commits/953a3120dd6c90a41892e3d9f937669269b0edce